x/ccv/provider/client/cli/tx.go

```diff
@@ -251,10 +251,11 @@ where create_consumer.json has the following structure:
     "top_N": 0,
     "validators_power_cap": 10,
     "validator_set_cap": 0,
-    "allowlist": ["cosmosvalcons..."],
```

Comment:
Why remove "cosmosvalcons..."? I added that intentionally to make it easier for the user to know the type of address needed.

Reply:
It was changed during the initial merge with main while working on this PR, but those changes should be reverted. It has been fixed now.